wa/processors/csvproc.py

```diff
@@ -77,8 +78,15 @@ def _write_results(self, results, output):
             writer.writerow(['id', 'workload', 'iteration', 'metric', ] +
                             extra_columns + ['value', 'units'])
 
-            for o in results:
-                header = [o.id, o.label, o.iteration]
+            for o in outputs:
+                if isinstance(o, JobOutput):
```

Comment:
this could just be `if o.kind == 'job':`

Reply:
Updated this. I also added an explicit check for kind=='job', just because it feels more rigorous.